src/scss/App.scss

```diff
@@ -0,0 +1,184 @@
+$background-color-main: #F4F4F4;
+$background-color-purple: #D0073E;
+$background-color-grey: #EDEDED;
+$text-dark-grey: #919191;
+$border-shadow: #D4D3D3;
+$border-dark-grey: #919191;
+$grey-three: #919191;
+$white: white;
+
+@font-face {
+  src: url('../assets/fonts/Abadi-MT-Condensed-Extra-Bold.ttf');
+  font-family: Abadi;
+}
+
+@font-face {
+  src: url('../assets/fonts/AvenirNextLTPro-Cn.otf');
+  font-family: Avenir;
+}
+
+@font-face {
+  src: url('../assets/fonts/nba-spurs.ttf');
+  font-family: spurs;
+}
+
+body {
+  margin: 0;
+  background-color: $background-color-main;
+}
+
+.app {
+  text-align: center;
+}
+
+.players {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.header-container {
+  display: flex;
+  height: 45px;
+  justify-content: space-evenly;
+  box-shadow: 1px -1px 2px $border-shadow;
+  font-family: Abadi;
+  list-style: none;
+  
+  .team {
+    position: relative;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background: $background-color-purple;
+
+    .logo-container {
+      display: flex;
+      border-radius: 24px;
+      box-shadow: 1px 1px 5px $border-dark-grey;
+      margin: 0 15px;
+      background-color: white;
+    }
+
+    button {
+      margin: 0 15px 0 0;
+      padding: 0;
```

Comment:
Remove `margin`, and give padding and height like so:

```
padding: 0 15px;
height: 100%;
```
This way the user can click the entire button area above, below and on the sides of the content too.